core/src/main/java/org/apache/accumulo/core/clientImpl/Namespaces.java

```diff
@@ -18,15 +18,10 @@
  */
 package org.apache.accumulo.core.clientImpl;
 
-import static java.nio.charset.StandardCharsets.UTF_8;
-
-import java.util.ArrayList;
 import java.util.LinkedList;
 import java.util.List;
 import java.util.Map.Entry;
 import java.util.SortedMap;
-import java.util.TreeMap;
-import java.util.function.BiConsumer;
```

Comment:
This comment is not for this line.  Later in this class there is an exists method.  That method could use the new NAmeSpaceMapping  object and it may be more efficient because it would do a map lookup instead of a linear scan of a list.

Reply:
Updated.